### DISCUSSION TITLE
How do you turn off system prompt for vision requests?

### DISCUSSION THREAD

**Discussion by Jonseed** [heshengtao/comfyui_LLM_party#108] [Q&A] [answered]
Some services with vision models don't accept a system prompt when you send an image in your request. They won't even accept an empty system prompt ("content": ""). I get this error in response:

`Error code: 400 - {'error': {'message': 'prompting with images is incompatible with system messages', 'type': 'invalid_request_error'}}`

How do you turn off system prompt entirely so it is not sent with the request?

**Comment by heshengtao** [comment, marked as answer]
I just improved it, and now if the system prompt word is empty, there will be no input of the system prompt word.

**Reply by Jonseed** [reply]
Thank you. It works now.
